tests/FSharp.Compiler.Private.Scripting.UnitTests/FSharpScriptTests.fs

```diff
@@ -143,7 +143,7 @@ System.Configuration.ConfigurationManager.AppSettings.Item "Environment" <- "LOC
     [<Theory>]
     [<InlineData("""#i " """,                                                                           // Single quote
                  "input.fsx (1,4)-(1,5) parse error End of file in string begun at or before here",
-                 "input.fsx (1,1)-(1,6) interactive warning Invalid directive '#i '")>]
+                 "input.fsx (1,1)-(1,3) interactive warning Invalid directive '#i '")>]
```

Comment:
This is a bug fix (see pars.fsy) for the range reported for bad hash errors.  Could be separated